src/lib.rs

```diff
@@ -1,10 +1,21 @@
+use game_state::SpatulaTier;
+use strum::EnumCount;
+
 pub mod game_state;
 pub mod lobby;
 pub mod net;
 pub mod player;
 
 pub const MAX_PLAYERS: usize = 6;
 
+pub struct GameRuleConsts {
+    pub spat_scores: [u32; SpatulaTier::COUNT],
+}
+
+pub const GAME_CONSTS: GameRuleConsts = GameRuleConsts {
+    spat_scores: [300, 100, 50, 0],
```

Comment:
Eventually we should make these configurable. Not going to stop this from merging. Its being tracked #58